bindata/egress-router/000-nad.yaml

```diff
@@ -13,13 +13,13 @@ spec:
     "name": "egress-router-cni-nad",
     "ip": {
       "addresses": [
-        "{{.Addresses}}"
+        "{{.Addresses}}/24"
         ],
       "destinations": [
-        "{{.AllowedDestinations}}"
+      {{range $i, $val := .AllowedDestinations}}{{if $i}},{{end}}{{if and ($val.Port) ($val.Protocol)}}{{if $val.TargetPort}}"{{$val.Port}} {{$val.Protocol}} {{$val.DestinationIP}}/30 {{$val.TargetPort}}"{{else}}"{{$val.Port}} {{$val.Protocol}} {{$val.DestinationIP}}/30"{{end}}{{else}}"{{$val.DestinationIP}}/30"{{end}}{{end}}
```

Comment:
This is parsing the format from allowedDestination field of the NAD that picks it up from the CRD value provided by the user which is in this format.

```
    redirectRules: [
      {
        destinationIP: "203.0.113.25",
        port: 65,
        protocol: TCP,
        targetPort: 75
      },
      {
        destinationIP: "203.0.113.26",
        port: 65535,
        protocol: SCTP,
      },
      {
        destinationIP: "203.0.113.27",
        port: 65534,
        protocol: UDP,
      },
    ],
    fallbackIP: "203.0.113.28"
```

What happens to the fallbackIP? I do not see that getting consumed?

Also I am not sure I follow why we add the `/30` to the destinationIP.